lms/djangoapps/program_enrollments/api/reading.py

```diff
@@ -477,14 +476,14 @@ def get_external_key_by_user_and_course(user, course_key):
     return relevant_pce.program_enrollment.external_user_key
 
 
-def get_saml_provider_by_org_key(org_key):
+def get_saml_providers_by_org_key(org_key):
     """
-    Returns the SAML provider associated with the provided org_key
+    Returns a list of SAML providers associated with the provided org_key
```

Comment:
In the comment, can we please add how multiple providers will be a possibility? Something like `With a single org, we might have multiple providers from multiple school, when the org_key is actually a combination of 2 or more schools. For example: RIT_MIT_x`
Something like the above goes a long way.

Reply:
That's also not the use case that motivated this change, but good to know that's a thing we support.